packages/react/src/TextInputWithTokens/TextInputWithTokens.tsx

```diff
@@ -79,7 +79,7 @@ function TextInputWithTokensInnerComponent<TokenComponentType extends AnyReactCo
     trailingVisual: TrailingVisual,
     loading,
     loaderPosition = 'auto',
-    contrast,
+    contrast, // deprecated
```

Comment:
I think we'd also want to add the `@deprecated` decorator to the `contrast` prop in the [StyledTextInputBaseWrapperProps](https://github.com/primer/react/blob/main/packages/react/src/internal/components/TextInputWrapper.tsx#L19) object.
Though I do see this is also used in `Select` in addition to `TextInput`,`TextArea` and `TextInputWithTokens`.
@mperrotti wondering if we're good to remove contrast for all of these or should we explore a different strategy? 🤔